### ISSUE TITLE
Getting "invalid syntax" errors - foreach loops, quietly, noisily, capture. 

### ISSUE THREAD

**Issue by danfridman** [kylebarron/stata_kernel#355]
#### Problem description

Using atom and hydrogen, I suddenly  get "invalid syntax r(198);" errors when executing code.

Context: I have been using stata_kernel with atom and hydrogen for nearly a year without this issue. This started happening after my computer ran out of hard disk space and STATA started throwing an error message. I deleted some unrelated files and cleared some space. That error disappeared, but now this error keeps popping up. I've updated the kernel, I're restarted everything many times and the problem is persistent. 

Only the most simple commands execute without a problem. I cannot use quietly, noisily or capture either without this error. 

![image](https://user-images.githubusercontent.com/45105185/93109968-b15c0280-f682-11ea-9c11-d30152787516.png)

There are no such problem when I launch STATA and use it directly. 

#### Debugging log

See attached. 
[console_debug.log](https://github.com/kylebarron/stata_kernel/files/5219692/console_debug.log)

Looking at it, it seems like a problem with include.do?

. `acf5410dffcb668adf37da696e87d400'
. _StataKernelLog on

. `d450cfde56d6cad2042c8f0870cc76f5'
. include C:/Users/Dan Fridman/.stata_kernel_cache/include.do
invalid syntax
r(198);

. `d1d0b5544eadcba44c5f1aa5695588da'
. display "test"
test

. `85a27f55e74d4469106bf93a0d482706'
. include C:/Users/Dan Fridman/.stata_kernel_cache/include.do
invalid syntax
r(198);

. `cc64df03024327541235dabb93f09e00'
. _StataKernelLog off

#### Code Sample

```stata
clear all

display "test"

foreach i in 1 2 3 4 {
	display "`i'"
}
```

**Comment by kylebarron** [comment]
@mcaceresb is this fixed with your most recent commit?

It's probably a good time to push another release to pypi...? I should probably figure out all the changes for the Changelog though.

**Comment by mcaceresb** [comment]
@kylebarron Should be.

@danfridman Can you try updating the kernel from the github repo? It should be working now.

**Comment by kylebarron** [comment]
(you should be able to install from github directly with
```bash
pip install -U git+https://github.com/kylebarron/stata_kernel
```

**Comment by danfridman** [comment]
@kylebarron Looks good! Thanks so much for your speedy response!

**Comment by kylebarron** [comment]
@mcaceresb I published 1.12.0

**Comment by kylebarron** [comment]
I also pushed an update to the docs site
